db/schema.rb

```diff
@@ -303,17 +303,17 @@
 
   create_table "statuses", force: :cascade do |t|
     t.integer "code"
-    t.string  "name", limit: 255
+    t.string  "name"
   end
 
   add_index "statuses", ["code"], name: "index_statuses_on_code", unique: true, using: :btree
   add_index "statuses", ["name"], name: "index_statuses_on_name", unique: true, using: :btree
 
   create_table "users", force: :cascade do |t|
-    t.string   "email",      limit: 255
+    t.string   "email"
     t.datetime "created_at"
     t.datetime "updated_at"
-    t.string   "user_name",  limit: 255
```

Comment:
If I remember correctly, these limits were added by default when we upgraded Rails. Looks like the db reset didn't add them again.